js/index.js

```diff
@@ -1 +1,72 @@
 // Your code goes here
+
+// NAVIGATION
+let logo = document.querySelector('.logo-heading');
+let navItems = document.querySelectorAll('.nav-link');
+
+const navigation = {
+  home: navItems[0],
+  aboutUs: navItems[1],
+  blog: navItems[2],
+  contact: navItems[3],
+}
+
+let isLogoClicked = false;
```

Comment:
I like this here. There will be a lot of stuff looking like this in the coming months